documentation/course/E1.md

```diff
@@ -113,7 +113,24 @@ W zakres projektu wchodzi:
 
 ### Płatność (Piotr Kot)
 
-- `PAY/01` - TODO@piterek130 - historyjki użytkownika
+- `PAY/01` - Jako _pasażer_ chcę mieć możliwość wyświetlenia listy dostępnych metod płatności.
+- `PAY/02` - Jako _pasażer_ chcę mieć możliwość dodania metody płatności BLIK do swojego _konta_.
+- `PAY/03` - Jako _pasażer_ chcę mieć możliwość dodania nowej karty płatniczej.
+- `PAY/04` - Jako _pasażer_ chcę mieć możliwość usunięcia karty płatniczej ze swojego _konta_.
+- `PAY/05` - Jako _pasażer_ chcę mieć możliwość usunięcia metody płatności ze swojego _konta_.
+- `PAY/06` - Jako _pasażer_ chcę mieć możliwość wyboru BLIK jako metodę płatności.
+- `PAY/07` - Jako _pasażer_ chcę mieć możliwość wyboru karty płatniczej jako metodę płatności. 
+- `PAY/08` - jako _pasażer_ chcę mieć możliwość dodania środków do swojego _portfela_.
+- `PAY/09` - jako _pasażer_ chcę mieć możliwość sprawdzenia stanu swojego _portfela_.
+- `PAY/10` - jako _pasażer_ chcę mieć możliwość przeglądania historii doładowań _portfela_.
+- `PAY/11` - jako _pasażer_ chcę mieć możliwość przeglądania historii płatności.
+- `PAY/12` - jako _pasażer_ chcę mieć możliwość zapłaty za _bilet_ za pomocą BLIK.
+- `PAY/13` - jako _pasażer_ chcę mieć możliwość zapłaty za _bilet_ za pomocą karty płatniczej.
+- `PAY/14` - jako _pasażer_ chcę mieć możliwość zapłaty za wystawiony _mandat_ za pomocą BLIK.
+- `PAY/15` - jako _pasażer_ chcę mieć możliwość zapłaty za wystawiony _mandat_ za pomocą karty płatniczej.
+- `PAY/16` - jako _bileter_ chcę mieć możliwość wystawienia _mandatu_.
+- `PAY/17` - jako _bileter_ chcę mieć możliwość anulowania _mandatu_.
```

Comment:
Some kind of status for fines like: `issued`, `paid`, `cancelled` would be useful

Reply:
This would need to be added to the dictionary, information model and business rules